### PR TITLE
V1

### PR DESCRIPTION
**Pull request by jwerle**
Lets get to version 1.0 🚀 

- [x] `bpkg.json` migration with `package.json` backwards compat
- [x] shellcheck and lint clean up
- [x] Actions CI

### CODE REVIEW

**Comment by francescobianco** [comment]
RESPECT

**Comment by jwerle** [comment]
@Potherca @francescobianco we have all checks passing now!

**Comment by Potherca** [comment]
I'll review this week.

**Review by Potherca — APPROVED** [review]
Other than one typo and 2 questions, I didn't see anything weird.

File: .shellcheckrc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+disable=SC2155
+
+## Allow: Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
+disable=SC2206
+
+## Allow: Prefer mapfile or read -a to split command output (or quote to avoid splitting)
+disable=SC2207
+
+## Allow: Instead of 'let expr', prefer (( expr ))
+disable=SC2219
```
[the remark this answers is not in the document]
Comment (jwerle) [reply .shellcheckrc:13]: 😂

**Review by Potherca — APPROVED** [review]

**Pull request merged**